src/main/java/org/reaktivity/nukleus/http2/internal/routable/stream/SourceInputStreamFactory.java

```diff
@@ -232,8 +234,8 @@ public MessageHandler newStream()
         int lastStreamId;
         long sourceRef;
         private long correlationId;
-        private final int initialWindow = 8192; // TODO config
-        int window = initialWindow;
+        private final int initialWindow = 65535; // TODO config
+        int window;
         int outWindow;
         int outWindowThreshold = -1;
```

Comment:
Yes, we use it in Http2WriteScheduler
```
    void flush()
    {
        if (connection.outWindow < connection.outWindowThreshold)
        {
            // Instead of sending small updates, wait until a bigger window accumulates
            return;
        }
```